packages/shorebird_cli/lib/src/shorebird_flutter.dart

```diff
@@ -60,4 +61,15 @@ class ShorebirdFlutter {
     );
     return status.isEmpty;
   }
+
+  Future<List<String>> getVersions({String? revision}) async {
+    final result = await git.forEachRef(
+      format: '%(refname:short)',
```

Comment:
Q: is this format string something we want the caller to be responsible for or should it be owned by `git`? lgtm either way, just curious.

Reply:
I left it as is for the sake of flexibility but happy to adjust in the future if we it's more convenient to keep it internal 👍